src/OSPSuite.Core/Domain/Services/SensitivityAnalyses/SensitivityAnalysisTask.cs

```diff
@@ -125,10 +100,7 @@ public bool Delete(IReadOnlyList<SensitivityAnalysis> sensitivityAnalyses)
 
       private void delete(SensitivityAnalysis sensitivityAnalysis)
       {
-         _applicationController.Close(sensitivityAnalysis);
```

Comment:
I remove the usage here so I need to react to the SensitivityAnalysisDeletedEvent somewhere else to close the view